templates/zerver/register.html

```diff
@@ -102,7 +102,20 @@ <h1>{{ _('Create your account') }}</h1>
                     <input type='hidden' name='key' value='{{ key }}' />
                     <input type='hidden' name='timezone' id='timezone'/>
                     <label for="id_email" class="inline-block label-title">{{ _('Email') }}</label>
-                    <div id="id_email">{{ email }}</div>
+                    <div class="email-visibility-text">
+                        {% if EMAIL_ADDRESS_VISIBILITY == EMAIL_ADDRESS_VISIBILITY_EVERYONE %}
+                        All users will be able to see this email address.
+                        {% elif EMAIL_ADDRESS_VISIBILITY == EMAIL_ADDRESS_VISIBILITY_ADMINS %}
+                        Only admin users will be able to see this email address.
+                        {% elif EMAIL_ADDRESS_VISIBILITY == EMAIL_ADDRESS_VISIBILITY_NOBODY %}
+                        Users will not be able to see this email address.
+                        {% else %}
+                        All users will be able to see this email address.
+                        {% endif %}
```

Comment:
I think it might be simpler to just have the backend computer and pass the text, in part because it'll let us write a handful of unit tests for the function generating the text, rather than having it all inline here.

Reply:
Sure thing. As a new contributor, when you say "backend computer" do you mean move this the python file as a function?

Reply:
Resolved in latest commit (6503c7f)

Reply:
Oops, yes, I meant "have the backend compute" -- and specifically have a Python function that returns this text and passes into the context used to render this template.

Reply:
Awesome! That's what I thought, thanks for your help!